content/blog/getting-started-with-volume-snapshots-on-a-kubernetes-cluster-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,840 @@
+
+
+
+```shell
+$ mysql -h 127.0.0.1 -uroot -pCfeDemo@123 -P 41797 -t <test_employees_sha.sql
++----------------------+
+| INFO                 |
++----------------------+
+| TESTING INSTALLATION |
++----------------------+
++--------------+------------------+------------------------------------------+
+| table_name   | expected_records | expected_crc                             |
++--------------+------------------+------------------------------------------+
+| departments  |                9 | 4b315afa0e35ca6649df897b958345bcb3d2b764 |
+| dept_emp     |           331603 | d95ab9fe07df0865f592574b3b33b9c741d9fd1b |
+| dept_manager |               24 | 9687a7d6f93ca8847388a42a6d8d93982a841c6c |
+| employees    |           300024 | 4d4aa689914d8fd41db7e45c2168e7dcb9697359 |
+| salaries     |          2844047 | b5a1785c27d75e33a4173aaa22ccf41ebd7d4a9f |
+| titles       |           443308 | d12d5f746b88f07e69b9e36675b6067abb01b60e |
++--------------+------------------+------------------------------------------+
++--------------+------------------+------------------------------------------+
+| table_name   | found_records    | found_crc                                |
++--------------+------------------+------------------------------------------+
+| departments  |                0 |                                          |
+| dept_emp     |                0 |                                          |
+| dept_manager |                0 |                                          |
+| employees    |           300024 | 4d4aa689914d8fd41db7e45c2168e7dcb9697359 |
+| salaries     |          2844047 | b5a1785c27d75e33a4173aaa22ccf41ebd7d4a9f |
+| titles       |           443308 | d12d5f746b88f07e69b9e36675b6067abb01b60e |
++--------------+------------------+------------------------------------------+
++--------------+---------------+-----------+
+| table_name   | records_match | crc_match |
++--------------+---------------+-----------+
+| departments  | not ok        | not ok    |
+| dept_emp     | not ok        | not ok    |
+| dept_manager | not ok        | not ok    |
+| employees    | OK            | ok        |
+| salaries     | OK            | ok        |
+| titles       | OK            | ok        |
++--------------+---------------+-----------+
++------------------+
+| computation_time |
++------------------+
+| 00:00:24         |
++------------------+
++---------+--------+
+| summary | result |
++---------+--------+
+| CRC     | FAIL   |
+| count   | FAIL   |
++---------+--------+
+```
+
+
+
+#### Perform MySQL database restore
+
+
+
+**1﻿. Scale MySQL database deployment config to 0**
+
+
+
+Before start MySQL database restore, I need to stop the mysql Pod by scaling the replicas in MySQL deployment to 0: 
+
+
+
+```markdown
+$ kubectl scale deployment.apps/mysql -n mysql --replicas=0
+deployment.apps/mysql scaled
+```
+
+
+
+T﻿ype below command to check the MySQL database deployment has 0 replica:
```

Comment:
Change to "Type the following command to check that the MySQL database deployment has 0 replica:"